---
layout: page
title: Speakers
permalink: /speakers/
---
<h3><a href="https://www.imperial.ac.uk/people/b.asquith">Becca Asquith</a> | The “stemness” of immune memory</h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Asquith.jpg" alt="Becca Asquith" style="float:Left;width:140px;padding-right:10px;"><b>Bio: </b>I use a novel combination of experimental, mathematical and bioinformatic techniques to investigate cell-mediated immunity. My aim is to develop a predictive mathematical model of the human in vivo CD8+ T cell response in order to prevent and alleviate viral infection. My initial training was in theoretical physics. I have a BSc in Physics (1st class hons. University College London), an MSc in Mathematics (distinction, Imperial College London) and a PhD in Theoretical Particle Physics (title: "Non Commutative Geometry", University of Durham). I then worked as a mathematical modeller for the UK government for 2 years before crossing disciplines to Mathematical Immunology. I held a Wellcome Trust Fellowship at Imperial College London and then a Leverhulme Trust Fellowship at the University of Oxford before returning to Imperial as a lecturer.
  </p>
<br>
<br>

<h3><a href="https://research.prinsesmaximacentrum.nl/en/team-members/patrick-kemmeren">Patrick Kemmeren</a> | Understanding regulatory systems and mechanisms of genetic interactions by exploiting expression profiles of large scale gene deletions</h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Kemmeren.PNG" alt="Patrick Kemmeren" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Bio: </b>
Patrick Kemmeren obtained his PhD in 2005 from Utrecht University for his work on analyzing and integrating genome-scale data. A postdoctoral fellowship obtained through EMBO and the Netherlands Genomics Initiative (NGI) allowed him to work at the University of California, San Francisco. Under the supervision of Dr. Nevan Krogan, he worked on the definition of an accurate and comprehensive map of stable protein-protein interactions. After returning to University Medical Center Utrecht (UMC Utrecht), he obtained a VENI grant from the Dutch Science Foundation (NWO) in 2007. After starting his own research group, Patrick received an NWO VIDI award in 2012 to investigate further the molecular mechanisms underlying genetic interactions. In 2013, Patrick became an Assistant Professor at the Center for Molecular Medicine, UMC Utrecht. In addition to these research activities, Patrick has been the coordinator of the High-Performance Computing facility for Life Sciences at Utrecht Science Park between 2012 and 2017, executive committee member of the Utrecht Bioinformatics Center since 2014 and board member of the Netherlands Bioinformatics and Systems Biology research school since 2017. Patrick joined the Princess Máxima Center for Pediatric Oncology as a group leader in 2016.  </p>
<br>
<br>

<h3><a href="https://www.ucl.ac.uk/physics-living-systems/training/msc-biological-physics/teaching-staff/dr-andela-saric">Andela Šarić</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Saric.png" alt="Andela Šarić" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Talk: </b><em>Title TBC</em><br><br><b>Bio: </b> Andela received her PhD from Columbia University in New York in 2013, after which she moved onto a postdoctoral fellowship at the University of Cambridge, where she was also a Research Fellow of Emmanuel College. She is currently a Junior Group Leader in the Department of Physics & Astronomy at UCL. Andela uses computer simulations to study biological assembly. She is interested in how proteins assemble “on-demand” into nanososcale structures that generate the molecular machinery of life. She is also devoted to understanding uncontrolled protein aggregation in the context of neurodegenerative diseases.


  </p>
<br>
<br>

<h3><a href="https://www.mpi-cbg.de/research-groups/current-groups/dora-tang/group-leader/">Dora Tang</a>
| Bottom up synthetic cellularity</h3>
  <p>
     <img src="/qsb2019/images/Tang.jpg" style="float:Left;width:140px;padding-right:10px;"><b>Biography: </b>T-Y Dora Tang received her PhD from Imperial College London, UK, in 2010 in the area of membrane biophysics. After 1 year as an EPSRC knowledge transfer secondee at Diamond Light Source, Oxfordshire, UK she undertook a post-doc at the University of Bristol, UK, in the areas of origin of life and then synthetic biology. In 2016 she started her independent lab at the MPI-CBG, Dresden as part of the MaxSynBio consortium. Her research lies between biophysics, synthetic biology and materials chemistry with the goal of building minimal synthetic cellular systems as physical models for biological phenomena specifically those related to compartmentalisation. These minimal systems aim to address questions in origin of life and in modern biology  such as “what are the conditions required to drive molecular organisation from disorder?”
  </p>
 <br>
 <br>

 
<h3><a href="https://www.microsoft.com/en-us/research/people/yordanov/">Boyan Yordanov</a> | TBC </h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/yordanov.jpg" alt="Boyan Yordanov" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Bio: </b>I am a scientist with the Biological Computation Group at Microsoft Research, working on programming biological systems and understanding the computation performed by living cells and organisms. My research aims to accelerate the design and reproducible construction of engineered biochemical circuits with robust, desirable behaviour (synthetic biology and DNA computing) and improve our understanding of the information processing strategies of natural biological systems (stem cell and developmental biology). To address these problems, I design, implement and apply methods that integrate computational analysis with experimental protocols. I was involved in setting up the first experimental biological lab at Microsoft Research, where we are now testing technologies that bring together computational analysis, experimental protocols, and lab automation to support our efforts in genetic engineering of living cells. I also contributed to the development of the RE:IN tool and methodology for reasoning about biological programs, which has been used to provide insights into the decision-making of stem cells.
  </p>
<br>
<br> 
<!--
<h3><a href="http://people.cryst.bbk.ac.uk/~ubcg71a/research.html">Irilenia Nobeli</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2018/images/nobeli.jpg" alt="Irilenia Nobeli" style="float:Left;width:140px;padding-right:10px;padding-bottom:30px;"><b>Talk: </b>From transcriptomic data to plausible biology hypotheses via bioinformatics<br><br><b>Bio: </b>Irilenia Nobeli's lab has expertise in computational biology and chemoinformatics. Current interests include regulatory RNAs, transcriptomics in health and disease, disorders of the brain and the role of small molecules in Biology.
  </p>
<br>
<br>
-->
