---
layout: page
title: Speakers
permalink: /speakers/
---
<h3><a href="https://www.imperial.ac.uk/people/b.asquith">Becca Asquith</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Asquith.jpg" alt="Becca Asquith" style="float:Left;width:140px;padding-right:10px;"><b>Talk: </b>Title TBC<br><br><b>Bio: </b>I use a novel combination of experimental, mathematical and bioinformatic techniques to investigate cell-mediated immunity. My aim is to develop a predictive mathematical model of the human in vivo CD8+ T cell response in order to prevent and alleviate viral infection. My initial training was in theoretical physics. I have a BSc in Physics (1st class hons. University College London), an MSc in Mathematics (distinction, Imperial College London) and a PhD in Theoretical Particle Physics (title: "Non Commutative Geometry", University of Durham). I then worked as a mathematical modeller for the UK government for 2 years before crossing disciplines to Mathematical Immunology. I held a Wellcome Trust Fellowship at Imperial College London and then a Leverhulme Trust Fellowship at the University of Oxford before returning to Imperial as a lecturer.
  </p>
<br>
<br>
<br>
<br>

<h3><a href="https://www.microsoft.com/en-us/research/people/sarajand/">Sara-Jane Dunn</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Dunn.jpg" alt="Sara-Jane Dunn" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Talk: </b>Title TBC<br><br><b>Bio: </b>I am a scientist based in the Biological Computation group at Microsoft Research Cambridge, and an Affiliate PI at the Wellcome-MRC Stem Cell Institute, University of Cambridge. I completed my undergraduate and postgraduate training at the University of Oxford, first obtaining a Masters in Mathematics, followed by a DPhil in Computational Biology from the Department of Computer Science. My research sits at the interface between mathematics, computer science and biology. More specifically, I focus on the concept of biology as computation, in which the biochemistry of the cell performs information processing in the service of decision-making. Towards this, I collaborate with experimentalists at the Universities of Cambridge and Padova to investigate stem cell decision-making throughout Development. As a computational scientist in biology, my goal is to identify the necessary abstractions to develop predictive, explanatory models of the biological programs that govern cellular behaviour. Ultimately, this research will lead us to make cells programmable, which could fundamentally transform medicine, agriculture and even the ways we generate energy.
  </p>
<br>
<br>

<h3><a href="https://research.prinsesmaximacentrum.nl/en/team-members/patrick-kemmeren">Patrick Kemmeren</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Kemmeren.PNG" alt="Patrick Kemmeren" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Talk: </b>Title TBC<br><br><b>Bio: </b>
Patrick Kemmeren obtained his PhD in 2005 from Utrecht University for his work on analyzing and integrating genome-scale data. A postdoctoral fellowship obtained through EMBO and the Netherlands Genomics Initiative (NGI) allowed him to work at the University of California, San Francisco. Under the supervision of Dr. Nevan Krogan, he worked on the definition of an accurate and comprehensive map of stable protein-protein interactions. After returning to University Medical Center Utrecht (UMC Utrecht), he obtained a VENI grant from the Dutch Science Foundation (NWO) in 2007. After starting his own research group, Patrick received an NWO VIDI award in 2012 to investigate further the molecular mechanisms underlying genetic interactions. In 2013, Patrick became an Assistant Professor at the Center for Molecular Medicine, UMC Utrecht. In addition to these research activities, Patrick has been the coordinator of the High-Performance Computing facility for Life Sciences at Utrecht Science Park between 2012 and 2017, executive committee member of the Utrecht Bioinformatics Center since 2014 and board member of the Netherlands Bioinformatics and Systems Biology research school since 2017. Patrick joined the Princess Máxima Center for Pediatric Oncology as a group leader in 2016.  </p>
<br>
<br>

<h3><a href="https://www.ucl.ac.uk/physics-living-systems/training/msc-biological-physics/teaching-staff/dr-andela-saric">Andela Šarić</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Saric.png" alt="Andela Šarić" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Talk: </b><em>Title TBC</em><br><br><b>Bio: </b> Andela received her PhD from Columbia University in New York in 2013, after which she moved onto a postdoctoral fellowship at the University of Cambridge, where she was also a Research Fellow of Emmanuel College. She is currently a Junior Group Leader in the Department of Physics & Astronomy at UCL. Andela uses computer simulations to study biological assembly. She is interested in how proteins assemble “on-demand” into nanososcale structures that generate the molecular machinery of life. She is also devoted to understanding uncontrolled protein aggregation in the context of neurodegenerative diseases.


  </p>
<br>
<br>

<h3><a href="https://www.mpi-cbg.de/research-groups/current-groups/dora-tang/group-leader/">Dora Tang</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2019/images/Tang.jpg" alt="Dora Tang" style="float:Left;width:140px;padding-right:10px;"><b>Talk: </b><em>Title TBC</em><br><br><b>Bio: </b> Biology is well equipped in exploiting a large number of out of equilibrium processes to support life. A complete understanding of these mechanisms is still in its infancy due to the complexity and number of the individual components involved in the reactions. However, a bottom up approach allows us to replicate key biological processes using a small number of basic building blocks. This methodology has the added advantage that properties and characteristics of the artificial cell can be readily tuned and adapted. Working between biophysics, materials science and synthetic biology we reimagine and translate the physical phenemona which drive out of equilibrium processes in cells into novel, robust and dynamic systems for synthetic biology applications. A key property of the cell is its ability to compartmentalize chemical reactions. This allows the cell to control different chemical and physical environments, utilize the membrane as a reaction surface and protect enzymes and proteins from degradation. Membrane delineated compartments based on lipids have been extensively used to fulfill this criteria, however they lack an internal heterogeneity that is characteristic of natural cells. Therefore, membrane free droplets based on coacervation or liquid-liquid phase separation have not only been associated with mechanisms within the natural cell but have offered an alternative model to compartmentalization. We use a range of techniques to understand the chemical and physical processes which drive molecular organization in lipids, polymers and proteins to rationally control self-assembly for the construction of novel proto-cellular platforms. This methodology is applied to protein-lipid and protein-polymer interactions as well as in-vitro compartmentalized transcription-translation processes which enables us to activate our compartments in a highly controlled manner. In addition, understanding these interactions can give insights to key questions in the origin of life i.e. what were the conditions required to drive molecular organization from disorder? and how did a biological world derive from chemistry?
  </p>
  
<!--
<h3><a href="http://people.cryst.bbk.ac.uk/~ubcg71a/research.html">Irilenia Nobeli</a></h3>
  <p>
     <img src="https://qsbworkshop.github.io/qsb2018/images/nobeli.jpg" alt="Irilenia Nobeli" style="float:Left;width:140px;padding-right:10px;padding-bottom:30px;"><b>Talk: </b>From transcriptomic data to plausible biology hypotheses via bioinformatics<br><br><b>Bio: </b>Irilenia Nobeli's lab has expertise in computational biology and chemoinformatics. Current interests include regulatory RNAs, transcriptomics in health and disease, disorders of the brain and the role of small molecules in Biology.
  </p>
<br>
<br>
-->
